src/App.tsx

```diff
@@ -1,24 +1,79 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  clockVisible: boolean;
+  nameOfClock: string;
 };
 
+class App extends React.Component<{}, State> {
+  state: State = {
+    clockVisible: true,
+    nameOfClock: 'Time to approve my task',
+  };
+
+  randomClockNames = ['Time to study JS', 'Time to relax', 'Time to study React', 'Time to think about where to celebrate the New Year'];
```

Comment:
Whoooa! A+ for this)

Reply:
Great idea, let's format it a bit
```suggestion
  randomClockNames = [
  	'Time to study JS',
  	'Time to relax',
  	'Time to study React', 
  	'Time to think about where to celebrate the New Year',
  ];
```